OpenKeychain/src/test/java/org/sufficientlysecure/keychain/pgp/KeyRingTest.java

```diff
@@ -30,28 +30,63 @@
 @Config(constants = WorkaroundBuildConfig.class, sdk = 21, manifest = "src/main/AndroidManifest.xml")
 public class KeyRingTest {
 
+    public static final String name = "Max Mustermann";
```

Comment:
I think it's easier and more readable to inline these vars, rather than piece them together with `String.format`